control/timeresp.py

```diff
@@ -458,8 +462,9 @@ def step_response(sys, T=None, X0=0., input=None, output=None,
     sys: StateSpace, or TransferFunction
         LTI system to simulate
 
-    T: array-like object, optional
-        Time vector (argument is autocomputed if not given)
+    T: array-like or number, optional
+        Time vector, or simulation time duration if a number (time vector is 
+        autocomputed if not given)
```

Comment:
Should we document someplace (perhaps in `doc/`) how this is autocomputed?

Reply:
I couldn't find an obvious place in doc about where to put it so I put in the docstring of step_response and referred back to that docstring in impuse, initial, and step_info.